### PR TITLE
[Redesign] Reset password

### PR DESCRIPTION
**Pull request by loic-sharma**
## Reset Password
reset1

## Password Reset Confirmation
reset2

@microsoftsam

### CODE REVIEW

File: src/NuGetGallery/Views/Users/PasswordChanged.cshtml [hunk 1]
```diff
@@ -1,10 +1,27 @@
...
\ No newline at end of file
+<section role="main" class="container main-container">
+    <div class="row">
+        <div class="col-sm-12">
+            <h1 class="text-center">Password changed!</h1>
+        </div>
+    </div>
+
+    <div class="row">
+        <div class="col-sm-offset-3 col-sm-6">
```
Comment (joelverhagen) [inline src/NuGetGallery/Views/Users/PasswordChanged.cshtml]: This is kinda a landing page. Would this look okay with 4 columns and centered>

File: src/NuGetGallery/Views/Users/PasswordChanged.cshtml [hunk 2]
```diff
@@ -1,10 +1,27 @@
...
+            @if (Request.IsAuthenticated)
+            {
+                <p>Your NuGet.org account password has been set and can now be used to sign in!</p>
```
Comment (scottbommarito) [inline src/NuGetGallery/Views/Users/PasswordChanged.cshtml]: `text-center` on this and the `<p>` below?

File: src/Bootstrap/less/theme/page-reset-password.less [hunk 3]
```diff
@@ -0,0 +1,5 @@
+.page-reset-password {
+  .reset-form {
+    margin-top: 80px;
```
Comment (scottbommarito) [inline src/Bootstrap/less/theme/page-reset-password.less]: I also see this on the Forgot Password form. We should move this to a shared location.

Comment (loic-sharma) [inline src/Bootstrap/less/theme/page-reset-password.less]: I don't think this is necessary for just two pages. If this becomes a common pattern, then sure, we can move it in a common location.

**Comment by loic-sharma** [comment]
Moved to a 4 column layout, centered the content's text, and added a missing error message (the old design was also missing this error message).

File: src/Bootstrap/less/theme/page-reset-password.less (hunk 3 above)
Comment (scottbommarito) [inline src/Bootstrap/less/theme/page-reset-password.less]: I think I used it on one of my other pages as well but I just referenced `forgot-form` directly.

File: src/NuGetGallery/Views/Users/ResetPassword.cshtml [hunk 4]
```diff
@@ -1,33 +1,59 @@
...
-    <p>
-        You can set a NuGet.org account password using the form below.
-    </p>
-}
+    <div class="row">
+        <div class="col-sm-offset-4 col-sm-4">
+            @if (ViewData.ModelState[""] != null && ViewData.ModelState[""].Errors.Count > 0)
+            {
+                @ViewHelpers.AlertDanger(
+                    @<text>@Html.ValidationMessage("")</text>
```
Comment (loic-sharma) [inline src/NuGetGallery/Views/Users/ResetPassword.cshtml]: This is to display the error message set on line 262 of `UsersController.cs` if the Password Reset Token is invalid or expired.

File: src/NuGetGallery/Views/Users/PasswordChanged.cshtml [hunk 5]
```diff
@@ -1,10 +1,27 @@
...
-    <p class="message">
-        Don't forget to use your new password the next time you
-        <a href="@Url.LogOn()">sign in</a>.
-    </p>
-}
\ No newline at end of file
+<section role="main" class="container main-container">
+    <div class="row">
+        <div class="col-sm-12">
+            <h1 class="text-center">Password changed!</h1>
```
Comment (ryuyu) [inline src/NuGetGallery/Views/Users/PasswordChanged.cshtml]: I think we have a 'page-heading' class that we've been using for the main heading text on a page. Is this something we would want to use here? Or should we consider this kind of centered heading a different category?

Comment (scottbommarito) [inline src/NuGetGallery/Views/Users/PasswordChanged.cshtml]: I was pretty sure `page-heading` was for the old site. I haven't used it on any of my new ones (at least intentionally).

File: src/NuGetGallery/Views/Users/PasswordChanged.cshtml [hunk 6]
```diff
@@ -1,10 +1,27 @@
...
+        </div>
+    </div>
+
+    <div class="row">
+        <div class="col-sm-offset-4 col-sm-4 text-center">
+            @if (Request.IsAuthenticated)
+            {
+                <p>Your NuGet.org account password has been set and can now be used to sign in!</p>
```
Comment (scottbommarito) [inline src/NuGetGallery/Views/Users/PasswordChanged.cshtml]: `text-center` here and below?

Comment (loic-sharma) [inline src/NuGetGallery/Views/Users/PasswordChanged.cshtml]: There's a `text-center` on the div up above.

**Pull request merged**